### DISCUSSION TITLE
Feedback on exercise submissions

### DISCUSSION THREAD

**Discussion by marfischb** [dodona-edu/feedback-and-support#27] [Questions - Vragen] [answered]
Hello,
I am a new user of dodona and I have a question on the exercise feedback.
I have submitted a correct answer to an exercise, but still get as feedback that the code I submitted is wrong.
The following message appears: "Error while evaluating testcase: the condition has length > 1".
Could somebody please explain to me why this error message appears?
Thank you in advance.
Kind regards,
Martine

**Comment by chvp** [comment]
Could you link your solution? It's probably an issue with the tests of the exercises, but it's difficult to investigate without looking at it.

**Reply by marfischb** [reply]
Thank you very much for your reply.
Here is a link: https://dodona.be/en/courses/335/series/3109/activities/1845156672/#
However, I am not sure if you will see my solution in this link. 

I also tried to attach a screenshot, but this did not work. If you can also help me with this, that would be great.

**Reply by chvp** [reply, marked as answer]
It turned out this was indeed an issue with certain R exercises. We fixed it on our end, so it should work now. (In the future, if you want to link a submission, right-click the submission number, e.g. #<wbr>2 in the following screenshot, and then copy the link.)

![screenshot_2023-12-04-100315](https://github.com/dodona-edu/dodona/assets/42220376/afb3f846-d250-45db-b310-a70ecbc729f1)

**Reply by marfischb** [reply]
Dear Charlotte,
Thank you very much for fixing it!
